package.json

```diff
@@ -21,8 +21,8 @@
     "url": "git+https://github.com/getredash/redash.git"
   },
   "engines": {
-    "node": "^8.0.0",
-    "npm": "^5.0.0"
+    "node": "^10.14.2",
+    "npm": "^6.4.1"
```

Comment:
Maybe worth keeping previous versions for now? Otherwise existing users will need to upgrade?

Reply:
I will revert this. Actually, these don't have to be updated, babel would transpile code well